src/js/index.js

```diff
@@ -1,37 +1,52 @@
+// import modules
 import 'reset-css';
 import '../scss/styles.scss';
 import scoreList from './modules/scoreList.js';
 import renderScores from './modules/renderScore.js';
+import fetchList from './modules/fetchList.js';
+import postScore from './modules/postScore.js';
 
-const scores = [
-  {
-    user: 'Name',
-    score: 100,
-  },
-  {
-    user: 'Name',
-    score: 20,
-  },
-  {
-    user: 'Name',
-    score: 50,
-  },
-  {
-    user: 'Name',
-    score: 78,
-  },
-  {
-    user: 'Name',
-    score: 125,
-  },
-  {
-    user: 'Name',
-    score: 77,
-  },
-  {
-    user: 'Name',
-    score: 42,
-  },
-];
-
-renderScores(scoreList(scores));
+// DOM elements
+const scoreForm = document.querySelector('.add__score-form');
+const refreshBtn = document.querySelector('.scoreboard__refresh-btn');
+
+// add listeners to the refresh button
+refreshBtn.addEventListener('click', () => {
+  // calling the api through the fetchList function
+  fetchList().then((data) => {
+    // calling the renderScores function to render the scores
+    renderScores(scoreList(data.result));
+    // update the local storage
+    localStorage.setItem('scores', JSON.stringify(data.result));
+  });
+});
```

Comment:
- [ ] The implementation of this button click is not rendering the data to the screen when the button is clicked. Kindly work on it so that when the button is clicked, the score list is shown on the page 👍

Reply:
> * [x]  The implementation of this button click is not rendering the data to the screen when the button is clicked. Kindly work on it so that when the refresh button is clicked, the score list is shown on the page 👍

Thanks! i didn't add the API key to the code. it has been fixed